.github/workflows/test-invoke-pip.yml

```diff
@@ -23,6 +23,7 @@ jobs:
         python-version:
           # - '3.9'
           - '3.10'
+          - '3.11'
```

Comment:
hmm, this means all the tests need to run twice - do we have any concerns about using up our runner minutes?

Reply:
Hmm good question - we can have tests for only 3.10 for the time being